src/index.js

```diff
@@ -43,6 +43,7 @@ export function isJSONfile(url) {
 export function transformJSONtoSass(json) {
   return Object.keys(json)
     .filter(key => isValidKey(key))
+    .filter(key => json[key] !== '#')
```

Comment:
Thanks again for the contributions and sorry for the delay! Can this be moved into the `isValidKey` regexp?

Reply:
`isValidKey` validates the key. The filter I've added validates the value.